Harrastuspassi/Controllers/HobbyDetails/HobbyDetailViewController.swift

```diff
@@ -62,6 +62,14 @@ class HobbyDetailViewController: UIViewController, UIScrollViewDelegate, UIGestu
         eventTableView.delegate = self;
         eventTableView.dataSource = self;
         linkActivityIndicator.isHidden = true;
+        let defaults = UserDefaults.standard;
+        let lat = defaults.float(forKey: DefaultKeys.Location.lat);
+        let lon = defaults.float(forKey: DefaultKeys.Location.lon);
+        let finland = GMSCameraPosition.camera(withLatitude: Double(lat),
```

Comment:
Mybe 'finland' is not optical name?

Reply:
Maybe, but it is also used here https://github.com/City-of-Helsinki/harrastuspassi-ui-ios/blob/master/Harrastuspassi/Controllers/Home/Map/MapViewController.swift#L47 so I wanted to keep it consistent